### ISSUE TITLE
leveldb

### ISSUE THREAD

**Issue by jainyzau** [conan-io/wishlist#25]
Dear Team,

Would you please provide this leveldb package? Thank you. Any recent version is OK. We are developing some financial programs, which are required to be low latency. So we us leveldb for data persistence. We are using it in both Windows and Linux:

Linux: CentOS6.7-GCC4.4.7-x64
Windows: VS2015 (Visual Studio 14.0) x64

The repo URL: https://github.com/google/leveldb

Thank you!

Br,
Jainy

**Comment by lasote** [comment]
How are you compiling it with Visual Studio?Thanks

**Comment by lasote** [comment]
Hi, I've uploaded leveldb for linux (don't know how or if it's possible to compile it in Windows, please tell me if you know it).
I also created and uploaded a package for Google performance tools (dependency of leveldb).
Please! give feedback! It's valid for you? Any other configuration/compiler/version needed?

leveldb/1.19@lasote/stable
gperftools/2.5@lasote/stable

**Comment by Dushistov** [comment]
@lasote 

> How are you compiling it with Visual Studio?Thanks

Officialy it is not supported, but there is fork that add cmake + windows support:

https://github.com/antmak/leveldb/tree/cmake-msvc

**Comment by jainyzau** [comment]
@lasote I'm so sorry for such a late reply, I don't use Github too much, and forget I have such a request, but now I do recall that. Thank you very much for your effort. Due to network reason, we didn't use conan for our projects, and from late last year, I have been working on Windows projects. We use our own built leveldb for our projects.

We are now considering constructing our private C++ package management system, and investigating `conan` and `hunter`. We will update to you if we use conan.

Regarding this request of leveldb, I will try it and give you feedback.

**Comment by jainyzau** [comment]
Due to well known reason in China, I can't install conan in my Ubuntu. I will try to use a VPS for testing.

**Comment by memsharded** [comment]
@jainyzau sorry for ignorance, what is being blocked? installs from PyPI? Could you maybe get one of the installers? Can you just git clone and run from sources? Please tell if we could help somehow.

**Comment by jainyzau** [comment]
I tried to download the installer, and tried to install it with pip, but both failed. Yes, thanks for reminding, I think I can try clone and run from sources.

**Comment by jainyzau** [comment]
@lasote @memsharded Finally, I made it. Conan itself and its packages on Bintray are not blocked by our great country, but some Python related stuffs are blocked by some ISPs. I changed PIP repository to a mirror in local and I can easily install Conan then. I also built leveldb via `conan install .. --build leveldb` and tested it on the Poco timer example, and everything just works! There's only one problem left: leveldb on Windows. But let's leave it as a homework for me. Thank you so much for your work.

![conan leveldb test](https://user-images.githubusercontent.com/4084365/29701603-36e050c2-899f-11e7-9dce-5a15a4498175.png)
